web/src/components/common/FormRow.tsx

```diff
@@ -36,7 +54,11 @@ const StyledCol = styled(Col)`
   align-items: left;
 `;
 
-const Label = styled.label`
+interface LabelProps {
+  stacked?: boolean;
+}
+
+const nonStackedLabelStyle = css`
```

Comment:
Is there a name we can use that doesn't use a negative? Hahaha Something like inlineLabel?

Reply:
@karenfrilya97 HAHAHA naming is hard! Sounds good, thanks!